### PR TITLE
VSTS - RepoName & ProjectName

### PR DESCRIPTION
**Pull request by jycouet**
VSTS has 3 level:
- endpoint
- project 
- repo

By default, repo and project has the same name.
In my VSTS implementation I didn't take care of project level! :(

Here is the issue link to RenovateMe (VSTS task based on renovate)
https://github.com/jycouet/VSTSExtensions/issues/7

With this PR, the project level will be handle! :)

### CODE REVIEW

File: lib/platform/vsts/index.js [hunk 1]
```diff
@@ -57,7 +57,10 @@ async function initRepo(repoName, token, endpoint) {
   config.fileList = null;
   config.prList = null;
   const repos = await gitApi().getRepositories();
-  const repo = repos.filter(c => c.name === repoName)[0];
+  const names = vstsHelper.getProjectAndRepo(repoName);
+  const repo = repos.filter(
+    c => c.name === names.repo && c.project.name === names.project
```
Comment (DavidParks8) [inline lib/platform/vsts/index.js]: This should probably be a case insensitive string compare.

File: test/platform/vsts/vsts-helper.spec.js [hunk 2]
```diff
@@ -298,4 +298,26 @@ describe('platform/vsts/helpers', () => {
       expect(res).toMatchSnapshot();
     });
   });
+
+  describe('getProjectAndRepo', () => {
+    it('should return the object with same strings', async () => {
```
Comment (DavidParks8) [inline test/platform/vsts/vsts-helper.spec.js]: If this is jasmine (I was lazy and didn't look), then you can speed up your tests by a significant amount by using the done callback inside async tests. 

```js
it('test', async (done) => {
  try {
    await something();
    done();
  } catch (error) {
    done.fail(error);
  }
});
```

Comment (rarkins) [reply test/platform/vsts/vsts-helper.spec.js]: we use jest

Comment (DavidParks8) [reply test/platform/vsts/vsts-helper.spec.js]: The same applies to jest as show [here](https://facebook.github.io/jest/docs/en/asynchronous.html)

Comment (DavidParks8) [reply test/platform/vsts/vsts-helper.spec.js]: Ah, got it.

File: lib/platform/vsts/index.js (hunk 1 above)
Comment (jycouet) [reply lib/platform/vsts/index.js]: I just checked in VSTS and yes we cannot create `repo1` and `Repo1`. So it make sense to check in lowerCase.

I'll update this tonight

**Comment by jycouet** [comment]
Ready 👍

**Comment by jycouet** [comment]
That was fast, thank you @rarkins.
How can we follow published version? @latest is sync with master?

**Pull request merged**